core/metamodel/src/main/java/org/apache/isis/core/runtime/threadpool/ThreadPoolSupport.java

```diff
@@ -143,16 +131,6 @@ public void close() throws Exception {
         return invokeAll(sequentialExecutor, callables);
     }
 
-    /**
-     * Executes specified {@code callables} on the sequential executor in sequence, one by one.
-     * @param callables nullable
-     * @return non-null
-     */
-    public List<Future<Object>> invokeAllSequential(final Callable<Object>... callables) {
```

Comment:
compiler warning: varargs possible heap pollution, so maybe just keep the API clean